<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HFormatSetDlg</class>
 <widget class="QDialog" name="HFormatSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>521</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>数字</string>
    </attribute>
    <widget class="QListWidget" name="catagoryListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>311</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>分类:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="title">
      <string>示例</string>
     </property>
    </widget>
    <widget class="QLabel" name="inforLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>说明</string>
     </property>
    </widget>
    <widget class="QListWidget" name="typeListWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>92</y>
       <width>371</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>基本</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>文本对齐方式</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>24</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>水平对齐:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="horizontalComboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="veritcalComboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>25</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>垂直对齐:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>471</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>文本控制</string>
     </property>
     <widget class="QCheckBox" name="autoWrapTextCheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>23</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>自动换行(&amp;W)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="mergeCellCheckBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>合并单元格(&amp;M)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>打印显示</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>491</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>票号格式设置</string>
     </property>
     <widget class="QCheckBox" name="prefixCheck">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>25</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>前缀</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prefixLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>23</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="suffixLineEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>23</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QCheckBox" name="suffixCheck">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>25</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>后缀</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="tempSheetcheckBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>54</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>临时票号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tempSheetLineEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>52</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="sheetLenLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>52</y>
        <width>130</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QCheckBox" name="sheetLencheckBox">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>54</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>票号长度</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>491</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>任务长度设置</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>操作任务(汉字/行)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="opTaskLineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>19</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="opItemLineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>19</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>操作项目(汉字/行)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serialNoLineEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>49</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>流水号长度</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="stChangeLineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>49</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>状态转换(汉字/行)</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>491</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>自定义显示</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>打印设置</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>4</y>
       <width>511</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>页边距</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>上：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="topMargin">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>22</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>22</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>下：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bottomMargin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>22</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="rightMargin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>左：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leftMargin">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>50</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>右：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tailMargin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="headMargin">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>80</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>尾：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>头：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>426</x>
        <y>103</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>单位:厘米</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>131</y>
       <width>511</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>页头和页尾</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>页头：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="headText">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>43</y>
        <width>491</width>
        <height>60</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>117</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>页尾：</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="tailText">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>140</y>
        <width>491</width>
        <height>60</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>其他设置</string>
     </property>
     <widget class="QCheckBox" name="gridCheck">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>23</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>网格线</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="rowCheck">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>23</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>显示行</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="clrColor">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>23</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>彩色打印</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="colCheck">
      <property name="geometry">
       <rect>
        <x>221</x>
        <y>23</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>显示列</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
